draft-ietf-quic-recovery.md

```diff
@@ -834,19 +834,19 @@ DetectLostPackets(largest_acked):
   foreach (unacked < largest_acked.packet_number):
     time_since_sent = now() - unacked.time_sent
     delta = largest_acked.packet_number - unacked.packet_number
-    if (time_since_sent > delay_until_lost):
-      lost_packets.insert(unacked)
-    else if (delta > reordering_threshold)
+    if (time_since_sent > delay_until_lost ||
+      delta > reordering_threshold):
       lost_packets.insert(unacked)
+      sent_packets.remove(unacked.packet_number)
+      if (!unacked.is_ack_only):
+        lost_packets.insert(unacked)
```

Comment:
You already inserted this packet 3 lines above.  Did you mean to remove that line?

Reply:
Yes. Fixed.